### PR TITLE
Fix test PATH variable and add challenge/task authorization

### PR DESCRIPTION
**Pull request by victorbalan**

### CODE REVIEW

File: platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Runner.java [hunk 1]
```diff
@@ -18,6 +18,10 @@
     @OneToMany(mappedBy = "runner")
     private List<Test> tests;
 
+    @OneToMany(mappedBy = "runner")
+    private List<Task> tasks;
+
+
```
Comment (flowlo) [inline platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Runner.java]: Two empty lines.

**Comment by flowlo** [comment]
LGTM

**Comment by victorbalan** [comment]
[3/8/2016 5:10:20 PM] Jakob Englisch: challenges are unique access organizations
[3/8/2016 5:10:32 PM] Jakob Englisch: and this logic should make the decision why XY can access some challenge
[3/8/2016 5:10:49 PM] Jakob Englisch: there is no need to send the organization id to this method?
[3/8/2016 5:11:50 PM] Jakob Englisch: why is it necessary the check the organization there?

so we know as what org is the user logged in

**Comment by Exordian** [comment]
removed the organization check for now and merged in smaller pieces

**Pull request closed without merging**